app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -20,6 +20,13 @@ fullwidth: true
 content:
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/listing
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/update-details
+inset_text:
```

Comment:
@emmaachesongray we should remove lines 23 to 29 for now given that you've removed the expander from appearing on the front end